tree/tree/inc/ROOT/InternalTreeUtils.hxx

```diff
@@ -61,18 +61,25 @@ struct RFriendInfo {
    */
    std::vector<std::vector<std::string>> fFriendChainSubNames;
 
-   void AddFriend(const std::string &treeName, const std::string &fileNameGlob, const std::string &alias = "");
+   /**
+    * Entries of each tree in each friend.
+    */
```

Comment:
can be just `/// Number of entries in each friend tree.`

Reply:
Rephrased to 
```
    * Number of entries contained in each tree of each friend. The outer
    * dimension of the vector tracks the n-th friend tree/chain, the inner
    * dimension tracks the number of entries of each tree in the current friend.
```